<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settingswindow</class>
 <widget class="QDialog" name="Settingswindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>534</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>570</width>
    <height>534</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>570</width>
    <height>534</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <widget class="QLabel" name="LBL_TITLE">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
  </widget>
  <widget class="QComboBox" name="CMBX_LNG">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>60</y>
     <width>191</width>
     <height>24</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>English</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Deutsch</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Nederlands</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Français</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Español</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Italiano</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>中文</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>日本語</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>한국어</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>العربية</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="LBL_LNG">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Language:</string>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_CUR">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>105</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Currency:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="CMBX_CUR">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>105</y>
     <width>191</width>
     <height>24</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Euro (€)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>US Dollar ($)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>British Pound (£)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Swiss Franc (CHF)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Japanese Yen (¥)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>South Korean Won (₩)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Australian Dollar (A$)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Canadian Dollar (C$)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>New Zeeland Dollar (NZ$)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Swedish Krona (kr)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Norwegian Krone (kr)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Polish Zloty (zł)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Czech Koruna (Kč)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Hungarian Forint (Ft)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Russian Ruble (₽)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Brazillian Real (R$)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Chinese Yuan (CN¥)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Mexican Peso (MXN$)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Singapore Dollar (S$)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Thai Baht (฿)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Hong Kong Dollar (HK$)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Philippine Peso (₱)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Romanian Leu (RON)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Icelandic Króna (kr)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Turkish Lira (₺)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Indonesian Rupiah (Rp)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Indian Rupee (₹)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Malaysian Ringgit (RM)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>South African Rand (R)</string>
    </property>
   </item>
  </widget>
  <widget class="Line" name="LN_HORZON">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>450</y>
     <width>551</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
  </widget>
  <widget class="QCheckBox" name="CB_DT">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>470</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Devtools enabled</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_ER">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>420</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Erase all data</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_RES">
   <property name="geometry">
    <rect>
     <x>369</x>
     <y>420</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset settings</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_SAVE">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>420</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_VER">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>10</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>App version: 0.0</string>
   </property>
  </widget>
  <widget class="QComboBox" name="CMBX_DATE">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>150</y>
     <width>191</width>
     <height>24</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>dd.mm.yyyy</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>mm.dd.yyyy</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>yyyy.mm.dd</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="LBL_DATE">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>81</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Date format:</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QSpinBox" name="SPBX_BU">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>195</y>
     <width>81</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>10</number>
   </property>
   <property name="value">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_MXB">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>175</y>
     <width>81</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Max. backups:</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="CB_EAS">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>105</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Enable autosave</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="CB_EL">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>470</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Enable logging</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="CB_CD">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>60</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Confirm on delete</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="CB_BOAE">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>150</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Backup on application exit</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="TBTN_DIR">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>370</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_FP">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>241</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Data file path:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LN_FLPT">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>370</y>
     <width>241</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
